packages/core/src/promotionEvaluations/redux/selectors.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @module promotionEvaluations/selectors
+ * @category Promotion Evaluations
+ * @subcategory Selectors
+ */
+
+import { getError, getIsLoading, getResult } from './reducer';
+
+/**
+ * Retrieves a list of all the promotion evaluations items available.
+ *
+ * @function
+ * @param {object} state - Application state.
+ * @returns {Array} - List of promotion evaluations items.
+ * @example
+ * import { getPromotionEvaluationItems } from '@farfetch/blackout-core/promotionEvaluations/redux';
+ *
+ * const mapStateToProps = state => ({
+ * promotionEvaluationsItems: getPromotionEvaluationItems(state)
+ * });
+ */
+export const getPromotionEvaluationItems = state =>
+  getResult(state.promotionEvaluations);
+
+/**
+ * Retrieves a list of all the promotion evaluations items available.
+ *
+ * @function
+ * @param {object} state - Application state.
+ * @param {string} id - Promotion evaluation item identifier to retrieve.
+ * @returns {object} - Promotion evaluations item.
```

Comment:
can you add some spaces before and after the `@returns`? I think it would look better :D

Reply:
The .find function can return undefined also :) Could you please update the `returns` description?

Reply:
Both done 🙂